packages/dds/tree/src/test/feature-libraries/editable-tree/editableTree.edit.spec.ts

```diff
@@ -0,0 +1,329 @@
+            trees[1].context.free();
+        });
+    })
+
+    describe("Sequences", () => {
+        it("Implicit sequence", async () => {
+            const expectedPhones = ["113", "114"];
+            const trees = await createSharedTrees(fullSchemaData, personData, 1);
+            const person = trees[0].root as PersonType;
+            assert(isEditableFieldSequence(person.address.sequencePhones));
+            for (let i = 0; i < person.address.sequencePhones.length; i++) {
+                assert.equal(person.address.sequencePhones[i], expectedPhones[i]);
+            }
+        });
+
+        it("update property", async () => {
+            const trees = await createSharedTrees(fullSchemaData, personData, 2);
+            const person1 = trees[0].root as PersonType;
+            const person2 = trees[1].root as PersonType;
+            const phones = person1.address.phones;
+            assert(isEditableFieldSequence(phones));
+            phones[1] = 123;
+            assert.equal(phones[1], 123);
+            assert.throws(() => {
+                phones[2] = { number: "123", prefix: "456" } as any;
+            });
+            trees[1].context.prepareForEdit();
+            await _provider.ensureSynchronized();
+            assert.deepEqual(person1, person2);
+            trees[0].context.free();
+            trees[1].context.free();
+        });
+    
+        it("add property", async () => {
+            const trees = await createSharedTrees(fullSchemaData, personData, 1);
+            const person = trees[0].root as PersonType;
+            assert(isEditableFieldSequence(person.address.simplePhones));
+            assert.equal(person.address.simplePhones[getTypeSymbol](undefined, true), "Test:SimplePhones-1.0.0")
+            person.address.simplePhones.push("999");
+            assert.equal(person.address.simplePhones[person.address.simplePhones.length - 1], "999");
+            const morePhones = ["1", "2"];
+            person.address.simplePhones.push(...morePhones);
+            const expectedSimplePhones = ["112", "999", "1", "2"];
+            for (let i = 0; i < person.address.simplePhones.length; i++) {
+                assert.equal(person.address.simplePhones[i], expectedSimplePhones[i]);
+            }
+            assert(isEditableFieldSequence(person.address.phones));
+            person.address.phones[appendNodeSymbol](singleTextCursor({ type: stringSchema.name, value: "new entry" }));
+            assert.equal(person.address.phones[3], "new entry");
+            trees[0].context.free();
+        });
+    
+        it("delete property", async () => {
+            const trees = await createSharedTrees(fullSchemaData, personData, 1);
+            const person = trees[0].root as PersonType;
+            delete person.address.phones;
+            assert.equal(person.address.phones, undefined);
+            assert.equal("phones" in person.address, false);
+            trees[0].context.free();
+        });        
```

Comment:
There are some trailing spaces here.
```suggestion
        });
```

Reply:
The trailing spaces are still there. Our linter configuration should assert against this but it doesn't seem to do that anymore. :-/
I'll look into it.

Reply:
Hm., weird, thanks. Fixed again.